roles/tobiko/tasks/configure-tobiko.yml

```diff
@@ -0,0 +1,31 @@
+- name: Get keystone data
+  register: keystone_data
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.command:
+    cmd: oc get keystoneapi keystone -n openstack -o json
+
+- name: Set keystone vars
+  ansible.builtin.set_fact:
+    keystone_secret_name: "{{ keystone_data.stdout | from_json | community.general.json_query('spec.secret') }}"
```

Comment:
json_query for this simple dict access is an overkill that will force the running host to have jmespath in place